ang/afform/afsearchSiteTokens.aff.json

```diff
@@ -0,0 +1,31 @@
+{
+    "type": "search",
+    "title": "Site Tokens",
+    "placement": [],
+    "icon": "",
+    "server_route": "civicrm/admin/sitetoken",
+    "permission": [
+        "edit system workflow message templates"
+    ],
+    "permission_operator": "AND",
+    "navigation": {
+        "parent": "Communications",
+        "label": "Site Tokens",
+        "weight": 10
+    },
+    "modified_date": "2024-06-16 15:50:36",
```

Comment:
seems a bit weird to include modified date here - but I would defer to @colemanw on that